ricecooker/utils/youtube.py

```diff
@@ -337,12 +338,16 @@ def get_language_with_alpha2_fallback(language_code):
     using the `getlang_by_alpha2` helper method.
     Returns either a le-utils Language object or None if both lookups fail.
     """
-    # 1. try to lookup `language` using internal representation
-    language_obj = languages.getlang(language_code)
-    # if language_obj not None, we know `language` is a valid language_id in the internal repr.
-    if language_obj is None:
-        # 2. try to match by two-letter ISO code
-        language_obj = languages.getlang_by_alpha2(language_code)
+    try:
+        language = langcodes.get(language_code)
+    except LookupError:
+        return
+    if not language.is_valid():
+        return
+    # try to lookup `language` using either the default tag or the alpha3 tag
+    language_obj = languages.getlang(language.to_tag()) or languages.getlang(
+        language.to_alpha3()
+    )
```

Comment:
Will this be returning `None` if the calls to getlang() goes sideways somehow?

Reply:
Yes, it returns None if no language matches the code.